{"name": "test_wpa_wpa2_enterprise_mixed_5g[setup_profiles0]", "status": "failed", "statusDetails": {"message": "AssertionError: assert False", "trace": "self = <tests.e2e.basic.validation_of_operating_modes.vlan_mode.client_connectivity.test_enterprise_ttls.TestvlanModeEnterpriseTTLSSuiteTwo object at 0x7fd4a028bd00>\nget_vif_state = ['ssid_wpa3_mixed_eap_2g', 'ssid_wpa3_mixed_eap_5g', 'ssid_wpa_wpa2_eap_2g', 'ssid_wpa_wpa2_eap_5g']\nstation_names_fiveg = ['wlan000']\nsetup_profiles = {'equipment_ap': True, 'radius_profile': True, 'vifc': True, 'vifs': True, ...}\nget_lanforge_data = {'lanforge-port-number': 8080, 'lanforge_2dot4g': 'wiphy0', 'lanforge_2dot4g_prefix': 'wlan0', 'lanforge_2dot4g_station': 'wlan0', ...}\nlf_test = <lanforge.lf_tests.RunTest object at 0x7fd4a82faca0>\nupdate_report = <testrails.reporting.Reporting object at 0x7fd4a93a4400>\ntest_cases = {'2g_open_bridge': 2234, '2g_open_nat': 4321, '2g_open_vlan': 9897, '2g_wpa2_bridge': 2237, ...}\nradius_info = {'ip': '10.10.10.72', 'name': 'basic-01-Automation-Radius-Profile-BRIDGE', 'password': 'password', 'pk_password': 'whatever', ...}\nexit_on_fail = False\n\n    @pytest.mark.wpa_wpa2_enterprise_mixed\n    @pytest.mark.fiveg\n    @allure.severity(allure.severity_level.CRITICAL)\n    def test_wpa_wpa2_enterprise_mixed_5g(self, get_vif_state,station_names_fiveg, setup_profiles, get_lanforge_data, lf_test,\n                                    update_report, test_cases, radius_info, exit_on_fail):\n        profile_data = setup_params_enterprise_two[\"ssid_modes\"][\"wpa_wpa2_enterprise_mixed\"][1]\n        ssid_name = profile_data[\"ssid_name\"]\n        security = \"wpa\"\n        extra_secu = [\"wpa2\"]\n    \n        band = \"fievg\"\n        vlan = 100\n        ttls_passwd = radius_info[\"password\"]\n        eap = \"TTLS\"\n        identity = radius_info['user']\n        if ssid_name not in get_vif_state:\n            allure.attach(name=\"retest,vif state ssid not available:\", body=str(get_vif_state))\n            pytest.xfail(\"SSID NOT AVAILABLE IN VIF STATE\")\n        passes = lf_test.EAP_Connect(ssid=ssid_name, security=security, extra_securities=extra_secu,\n                                     mode=mode, band=band,\n                                     eap=eap, ttls_passwd=ttls_passwd, identity=identity,\n                                     station_name=station_names_fiveg, vlan_id=vlan)\n    \n        if passes:\n            update_report.update_testrail(case_id=test_cases[\"5g_wpa2_mixed_eap_ttls_wpa2_vlan\"],\n                                          status_id=1,\n                                          msg='5G WPA2 Mixed Enterprise Client Connectivity Passed successfully - vlan mode' + str(\n                                              passes))\n        else:\n            update_report.update_testrail(case_id=test_cases[\"5g_wpa2_mixed_eap_ttls_wpa2_vlan\"],\n                                          status_id=5,\n                                          msg='5G WPA2 Mixed Client Connectivity Failed - vlan mode' + str(\n                                              passes))\n            if exit_on_fail:\n                pytest.exit(\"Test Case Failed\")\n>       assert passes\nE       assert False\n\ne2e/basic/validation_of_operating_modes/vlan_mode/client_connectivity/test_enterprise_ttls.py:361: AssertionError"}, "attachments": [{"name": "wlan000", "source": "1609a79b-838b-4fc8-8b16-f6723fddb252-attachment.attach"}, {"name": "cx_data", "source": "95142a30-8d6c-4e23-8874-4177c7112bd0-attachment.attach"}, {"name": "5G WPA2 Mixed Client Connectivity Failed - vlan modeFalse", "source": "0b595aa0-c16a-45c8-945d-0100076b8cd2-attachment.attach"}], "parameters": [{"name": "setup_profiles", "value": "{'mode': 'VLAN', 'ssid_modes': {'wpa_wpa2_enterprise_mixed': [{'ssid_name': 'ssid_wpa_wpa2_eap_2g', 'appliedRadios': ['is2dot4GHz'], 'profile_name': 'basic-01-SSID-wpa_wpa2_enterprise_mixed-0-VLAN', 'mode': 'BRIDGE', 'vlan': 100}, {'ssid_name': 'ssid_wpa_wpa2_eap_5g', 'appliedRadios': ['is5GHzU', 'is5GHz', 'is5GHzL'], 'profile_name': 'basic-01-SSID-wpa_wpa2_enterprise_mixed-1-VLAN', 'mode': 'BRIDGE', 'vlan': 100}], 'wpa3_enterprise_mixed': [{'ssid_name': 'ssid_wpa3_mixed_eap_2g', 'appliedRadios': ['is2dot4GHz'], 'profile_name': 'basic-01-SSID-wpa3_enterprise_mixed-0-VLAN', 'mode': 'BRIDGE', 'vlan': 100}, {'ssid_name': 'ssid_wpa3_mixed_eap_5g', 'appliedRadios': ['is5GHzU', 'is5GHz', 'is5GHzL'], 'profile_name': 'basic-01-SSID-wpa3_enterprise_mixed-1-VLAN', 'mode': 'BRIDGE', 'vlan': 100}]}, 'rf': {}, 'radius': True}"}], "start": 1626924744752, "stop": 1626924938459, "uuid": "0e805fb9-d27a-44b8-bf0f-efa5ff412256", "historyId": "491b69bcf55380b5e42912563ead13e6", "testCaseId": "c874e95d663ce317958550fd3caa2aa4", "fullName": "e2e.basic.validation_of_operating_modes.vlan_mode.client_connectivity.test_enterprise_ttls.TestvlanModeEnterpriseTTLSSuiteTwo#test_wpa_wpa2_enterprise_mixed_5g", "labels": [{"name": "severity", "value": "critical"}, {"name": "tag", "value": "enterprise"}, {"name": "tag", "value": "client_connectivity"}, {"name": "tag", "value": "vlan"}, {"name": "tag", "value": "sanity"}, {"name": "tag", "value": "@pytest.mark.usefixtures('setup_profiles')"}, {"name": "tag", "value": "ttls"}, {"name": "tag", "value": "wpa_wpa2_enterprise_mixed"}, {"name": "tag", "value": "fiveg"}, {"name": "parentSuite", "value": "e2e.basic.validation_of_operating_modes.vlan_mode.client_connectivity"}, {"name": "suite", "value": "test_enterprise_ttls"}, {"name": "subSuite", "value": "TestvlanModeEnterpriseTTLSSuiteTwo"}, {"name": "host", "value": "testing-87-ftvrb"}, {"name": "thread", "value": "6-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "e2e.basic.validation_of_operating_modes.vlan_mode.client_connectivity.test_enterprise_ttls"}]}